build.sbt

```
@@ -39,6 +39,16 @@ lazy val core: Project =
     folder    = "."
   )
 
+lazy val xpath: Project =
+  buildModule(
+    path      = "xpath",
+    toPublish = true,
+    folder    = "."
+  ).dependsOn(core % "test->test;compile->compile")
+    .settings(
+      libraryDependencies ++= Dependencies.parser
```

Check the comment about `Dependencies`. 

```suggestion
      libraryDependencies ++= Dependencies.XPath.dedicated
```